test/dom/serializer.test.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-const { DOMParser } = require('../../lib/dom-parser')
+const { DOMParser } = require('../../lib')
 const { XMLSerializer } = require('../../lib/dom')
```

Comment:
nit: They could be merged now
```suggestion
const { DOMParser, XMLSerializer } = require('../../lib')
```